goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["actor-hash-h",                               "actor-hash-h",               3, ["ENGINE", "GAME"], "engine/data"],
+["load-dgo",                                   "load-dgo",                   3, ["ENGINE", "GAME"], "engine/load"],
+["ramdisk",                                    "ramdisk",                    3, ["ENGINE", "GAME"], "engine/load"],
+["gsound",                                     "gsound",                     3, ["ENGINE", "GAME"], "engine/sound"],
+["transformq",                                 "transformq",                 3, ["ENGINE", "GAME"], "engine/math"],
+["collide-func",                               "collide-func",               3, ["ENGINE", "GAME"], "engine/collide"],
+["joint",                                      "joint",                      3, ["ENGINE", "GAME"], "engine/anim"],
+["joint-mod",                                  "joint-mod",                  3, ["ENGINE", "GAME"], "engine/anim"],
+["chain-physics",                              "chain-physics",              3, ["ENGINE", "GAME"], "engine/physics"],
+["cylinder",                                   "cylinder",                   3, ["ENGINE", "GAME"], "engine/geometry"],
+["wind-work",                                  "wind-work",                  3, ["ENGINE", "GAME"], "engine/gfx"],
+["wind",                                       "wind",                       3, ["ENGINE", "GAME"], "engine/gfx"],
+["bsp",                                        "bsp",                        3, ["ENGINE", "GAME"], "engine/gfx/vis"],
+["subdivide",                                  "subdivide",                  3, ["ENGINE", "GAME"], "engine/gfx/tfrag"],
+["sprite",                                     "sprite",                     3, ["ENGINE", "GAME"], "engine/gfx/sprite"],
+["sprite-distort",                             "sprite-distort",             3, ["ENGINE", "GAME"], "engine/gfx/sprite"],
+["sprite-glow",                                "sprite-glow",                3, ["ENGINE", "GAME"], "engine/gfx/sprite"],
+["debug-sphere",                               "debug-sphere",               3, ["ENGINE", "GAME"], "engine/debug"],
+["debug",                                      "debug",                      3, ["ENGINE", "GAME"], "engine/debug"],
+["history",                                    "history",                    3, ["ENGINE", "GAME"], "engine/data"],
+["merc-vu1",                                   "merc-vu1",                   3, ["ENGINE", "GAME"], "engine/gfx/merc"],
+["emerc-vu1",                                  "emerc-vu1",                  3, ["ENGINE", "GAME"], "engine/gfx/emerc"],
+["merc-blend-shape",                           "merc-blend-shape",           3, ["ENGINE", "GAME"], "engine/gfx/merc"],
+["merc",                                       "merc",                       3, ["ENGINE", "GAME"], "engine/gfx/merc"],
+["emerc",                                      "emerc",                      3, ["ENGINE", "GAME"], "engine/gfx/emerc"],
```

Comment:
probably just put in `gfx/merc` - the `emerc` renderer is merc with envmap and it seems to share a _ton_ of code/format with merc.